tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_Tov.cpp

```diff
@@ -108,25 +108,47 @@ void test_tov(
         custom_approx(interpolated_log_enthalpy_ds));
 }
 
-void test_tov_dp_dr(
-    const std::unique_ptr<EquationsOfState::EquationOfState<true, 1>>&
-        equation_of_state) {
-  const gr::Solutions::TovSolution radial_tov_solution(*equation_of_state,
-                                                       1.0e-3);
+void test_tov_dp_dr(const EquationsOfState::EquationOfState<true, 1>& eos) {
```

Comment:
It's not obvious to me why you rewrote this test. Can you please explain?

Reply:
I moved the function that computes the pressure and all the other variables that might be needed by the executables to RelEuler::TovStar, so the radial solution class now only does the ODE integration and exposes the result, from which everything else can be computed. This decouples the ODE integration from the system-specific analytic solution class. I could move this particular test also to RelEuler::TovStar, but I thought it's really a test of the ODE integration so it's better located here. Therefore I just compute the pressure from the log enthalpy and compare its FD derivative to the standard dp/dr TOV equation. Does that make sense?

Reply:
Yep, it does. Thank you!